pkg/operator/starter.go

```diff
@@ -129,15 +152,23 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		)
 	}
 
+	var snapshotAssetNames []string
+
+	if csiOperatorControllerConfig.SnapshotAssetNameFunc != nil {
+		snapshotAssetNames = csiOperatorControllerConfig.SnapshotAssetNameFunc()
+	} else {
+		snapshotAssetNames = a.GetVolumeSnapshotClassAssetNames()
+	}
```

Comment:
Instead of adding this if-then-else: could we ensure SnapshotAssetNameFunc is never nil and call it unconditionally? Then the aws operator config for example would just initialize `SnapshotAssetNameFunc = a.GetVolumeSnapshotClassAssetNames()`